modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -221,61 +321,66 @@ const getDriverSpec = (settings: DriverSettings, browserName: string): DriverLoa
  * webdriverTimeout: how long to wait for the webdriver server to start
  */
 export const create = async (settings: DriverSettings): Promise<Driver> => {
-
   const webdriverTimeout = settings.webdriverTimeout || 30000;
 
   const browserName = browserVariants[settings.browser] || settings.browser;
-
-  const driverSpec = getDriverSpec(settings, browserName);
-
-  try {
-    // Find an open port to start the driver on
-    const port = await getPort(settings.webdriverPort, 4444);
-    const debuggingPort = settings.headless ? await getPort(settings.debuggingPort, 9000) : 9000;
-
-    // Wait for the driver to start up and then start the webdriver session
-    await DriverLoader.startAndWaitForAlive(driverSpec, port, webdriverTimeout);
-
-    const webdriverOptions = getOptions(port, browserName, settings, debuggingPort);
-
-    if (settings.verbose) {
-      console.log(
-        `Browser capabilities: ${JSON.stringify(webdriverOptions.capabilities)}`
-      );
-    }
-
+  // Find an open port to start the driver on
+  const port = await getPort(settings.webdriverPort, 4444);
+  const debuggingPort = settings.headless ? await getPort(settings.debuggingPort, 9000) : 9000;
+  const webdriverOptions = getOptions(port, browserName, settings, debuggingPort);
+  console.log('Webdriver options:', webdriverOptions);
```

Comment:
Does these options need to be formatted or does it print it in a nice way?

Reply:
They get printed nicely in node. Only down to one level which should be good enough for shallow issues